lib/purchases_flutter.dart

```diff
@@ -159,8 +163,15 @@ class Purchases {
   /// [customerInfoUpdateListener] CustomerInfo update listener.
   static void addCustomerInfoUpdateListener(
     CustomerInfoUpdateListener customerInfoUpdateListener,
-  ) =>
-      _customerInfoUpdateListeners.add(customerInfoUpdateListener);
+  ) {
+    _customerInfoUpdateListeners.add(customerInfoUpdateListener);
+    if (_customerInfoUpdateListeners.length == 1) {
+      final lastReceivedCustomerInfo = _lastReceivedCustomerInfo;
+      if (lastReceivedCustomerInfo != null) {
+        customerInfoUpdateListener(lastReceivedCustomerInfo);
+      }
+    }
+  }
```

Comment:
Also note that this is actually a behavior change, so it should be prominently displayed in the changelog.

Reply:
yes, thankfully it said in the docs that this was the expected behavior, so I think everyone was assuming this was the behavior anyways